pyvista/utilities/__init__.py

```diff
@@ -1,38 +1,70 @@
-"""Deprecated utilities module."""
+"""Deprecated utilities subpackage."""
 import importlib
 import inspect
 import warnings
 
-from pyvista.core.errors import DeprecationError, PyVistaDeprecationWarning
+# Places to look for the utility
+_MODULES = [
+    'pyvista.core.utilities.cells',
```

Comment:
@akaszynski why is `pyvista.core.utilities.cells` added here? IMO, we should not include this and encourage users to use `from pyvista.core.utilities import <feature>` instead of going down into individual modules in the utilities subpackage.

Reply:
If something in the `cells.py` module is missing at the top level `pyvista.core.utilities` package we should make sure to include it in `pyvista/core/utilities/__init__.py`

Reply:
```suggestion
```

Reply:
I added this as one of the imports in a closed source project of mine was raising an `ImportError`, specifically `numpy_to_idarr`.

This was the import warning I was hoping to get:
```
/home/alex/source/pyvista/pyvista/utilities/_getattr.py:74: PyVistaDeprecationWarning: The `pyvista.utilities` module has been deprecated. `numpy_to_idarr` is now imported as: `from pyvista.core.utilities.cells import numpy_to_idarr`.
```

Reply:
IMO, this should be `from pyvista.core.utilities import numpy_to_idarr` to ease the pain if we ever re-organize these modules in `pyvista.core.utilities`